### PR TITLE
Fix UI clipping bug on mid size screens

### PR DESCRIPTION
**Pull request by ivanng-oursky**
## Overview page fully responsive

https://user-images.githubusercontent.com/106721852/178892905-3d719c35-d78b-4ca3-9b38-ff282d22a078.mov

### Portfolio page fully responsive

https://user-images.githubusercontent.com/106721852/178893376-bb8eac86-d5f2-4325-b665-d910908db49c.mov

## No overflow for proposals with super long h1 tag

image

refs oursky/likedao#244

### CODE REVIEW

**Comment by ivanng-oursky** [comment]
@hochiw updated 🙇 (see screens in description)

File: react-app/src/components/AppSideBar/AppSideBar.tsx [hunk 1]
```diff
@@ -206,7 +206,7 @@ const AppSideBar: React.FC<AppSideBarProps> = ({
           closeMobileMenu={onMenuClose}
         />
       </div>
-      <main className={cn("grow", "px-3", "sm:px-0")}>{children}</main>
+      <main className="grow px-3 min-w-0 sm:px-0">{children}</main>
```
Comment (hochiw) [inline react-app/src/components/AppSideBar/AppSideBar.tsx:209]: why did we remove the `cn` tag?

File: react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx [hunk 2]
```diff
@@ -62,9 +62,10 @@ export const CommunityStatusRegular: React.FC<CommunityStatusRegularProps> = (
         "flex",
         "flex-col",
         "gap-y-3",
-        "sm:gap-y-0",
+        "sm:gap-y-3",
```
Comment (hochiw) [inline react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx:65]: would this create extra spacing vertically on desktop view?

also there's `"gap-y-3"` already so no need to put it again

File: react-app/src/components/PortfolioScreen/PortfolioPanel.tsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ const CoinsAmountField: React.FC<{
   amount: BigNumber;
 }> = ({ messageID, amount }) => {
   return (
-    <div className={cn("flex", "flex-col", "mr-6")}>
+    <div className="flex flex-col sm:mb-4 sm:w-1/3 lg:mr-6 lg:w-auto">
```
Comment (hochiw) [inline react-app/src/components/PortfolioScreen/PortfolioPanel.tsx:72]: where did the `w-1/3` and `mb-4` come from

also there should only be one breakpoint - `sm`, shouldn't need special handling for an extra breakpoint

**Review by hochiw — CHANGES_REQUESTED** [review]

File: react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx (hunk 2 above)
Comment (ivanng-oursky) [reply react-app/src/components/CommunityStatus/CommunityStatusRegular.tsx:65]: > would this create extra spacing vertically on desktop view?

seems no

image

File: react-app/src/components/AppSideBar/AppSideBar.tsx (hunk 1 above)
Comment (ivanng-oursky) [reply react-app/src/components/AppSideBar/AppSideBar.tsx:209]: eh no reason, let me add it back

File: react-app/src/components/PortfolioScreen/PortfolioPanel.tsx (hunk 3 above)
Comment (ivanng-oursky) [reply react-app/src/components/PortfolioScreen/PortfolioPanel.tsx:72]: as discussed offline, will use grid instead

**Comment by ivanng-oursky** [comment]
updated your portfolio panel to overflow only content (fixed title)

image

**Pull request merged**